src/components/Community/Hero/index.tsx

```diff
@@ -17,12 +17,7 @@ const Hero: React.FC = () => {
 
   return (
     <LayoutWidthContainer className={styles.container}>
-      <Link
-        className={styles.link}
-        href={data.hero.url}
-        target="_blank"
-        onClick={logHero}
-      >
+      <Link className={styles.link} href={data.hero.url} onClick={logHero}>
```

Comment:
@rogermparent what will happen if put an external link in the next banner?

Reply:
Our `Link` wrapper component detects external links and adds proper `target="_blank"` and `rel="noopener noreferrer"` accordingly. Explicitly setting these props isn't necessary unless there's an edge case like wanting an internal link to have `target="_blank"`.